### PR TITLE
Function reformatting vector_norm

### LINKED ISSUE #10311: vector_norm

### PR DESCRIPTION
**Pull request by nikifaets**
Close #10311

### CODE REVIEW

**Comment by nikifaets** [comment]
Close #10311

**Comment by guillesanbri** [comment]
Hi! Thank you for the contribution! As this is a formatting task, please, follow the steps here https://lets-unify.ai/ivy/contributing/open_tasks.html#formatting-checklist :slightly_smiling_face: Thanks!

**Comment by nikifaets** [comment]
add_reformatting_checklist_linear_algebra

**Comment by nikifaets** [comment]
@guillesanbri  I opened the bug report for the numpy backend case: #10480  

I believe this PR is ready for review.

Regarding the SOS for the docstring tests:
I see they are not run as a CI job. If I understand correctly the config in `test-docstrings.yml`, the PR needs the `Function Reformatting` label to run the job?

However, I ran the docstring tests locally. In the logs I see the tests only check the **backend docstrings** which does not cover any of my changes. Am I missing something on how to run the docstrings tests with 100% coverage or specifically for my files/methods?

Here's the local docstring test summary:

```
=========================== short test summary info ============================
FAILED ivy_tests/test_docstrings.py::test_docstrings[jax] - AssertionError: 
FAILED ivy_tests/test_docstrings.py::test_docstrings[numpy] - AssertionError: 
FAILED ivy_tests/test_docstrings.py::test_docstrings[tensorflow] - AssertionE...
FAILED ivy_tests/test_docstrings.py::test_docstrings[torch] - AssertionError: 
======================= 4 failed, 35 warnings in 11.15s ========================
```

Note that the failing tests are for methods that aren't changed by this PR.

Thank you.

File: ivy/functional/ivy/linear_algebra.py [hunk 1]
```diff
@@ -2401,6 +2401,44 @@ def vector_norm(
...
+    >>> y = ivy.empty(shape = 1)
+    >>> y
+    ivy.array([0.  , 2.  , 0.  , 2.25])
+    >>> ivy.vector_norm(x, ord = 0, out = y)
+    ivy.array([4.])
+    >>> y
+    ivy.array([4.])
+
+    >>> x = ivy.arange(8).reshape((2,2,2))
+    >>> ivy.vector_norm(axis = (0,1), ord = float("-inf"))
```
Comment (guillesanbri) [inline ivy/functional/ivy/linear_algebra.py:2432]: seems like `x` is missing here :slightly_smiling_face:

File: ivy/functional/ivy/linear_algebra.py [hunk 2]
```diff
@@ -2401,6 +2401,44 @@ def vector_norm(
...
+    ivy.array([2, 4])
+
+    >>> x = ivy.Container(a = [-1., 1., -2., 2.], b = [0., 1.2, 2.3, -3.1])
+    >>> ivy.vector_norm(ord = -1)
```
Comment (guillesanbri) [inline ivy/functional/ivy/linear_algebra.py:2436]: same as above

**Comment by guillesanbri** [comment]
the CI seems to hung up when running the docstring tests, do they work fine in your local env?

**Comment by nikifaets** [comment]
@guillesanbri  Locally, they are failing for the same functions that fail on master.

**Comment by guillesanbri** [comment]
ah, I see, took me a bit haha seems like you are using `print` explicitly on the output, which is what the test looks for

**Comment by nikifaets** [comment]
I totally missed this too. Updated the examples.

**Comment by guillesanbri** [comment]
looks good to me now! I've seen your two messages on the discord server, given that it's something on our side I'll merge this. Thanks for the contribution!

**Pull request merged**